src/Month.js

```diff
@@ -89,7 +90,13 @@ class MonthView extends React.Component {
           scrollableMonth && 'rbc-month-view-infinite'
         )}
       >
-        <div className="rbc-row rbc-month-header">
+        <div
+          className="rbc-row rbc-month-header"
+          style={{
+            'margin-right': `${scrollableMonth ? scrollbarSize() - 1 : 0}px`,
+            'border-right': '1px solid #DDD',
```

Comment:
I think we should also only show this border-right if `scrollableMonth` is true, otherwise, we'll aways show this border-right even when there isn't a scrollbar. 

We could prob just do the ternary for the entire style object so it would be something like:

```
const style = scrollableMonth ? { 'margin-right': `${scrollbarSize() - 1}px`, 'border-right': '1px solid #DDD' } : {}

...
style={style}
...
```